### ISSUE TITLE
Update gradle build script to match my latest system. 

### ISSUE THREAD

**Issue by darkhax** [Darkhax-Minecraft/Enchanting-Plus#83]
This mod is using an older version of my gradle setup. I would appreciate it if someone could update it to match my latest gradle setup. Look at Bookshelf or Dark Utilities for examples of said setup. Should mostly be a copy paste job, along with updating the curse project id, and build dependencies.

**Comment by AshanthaLahiru** [comment]
What do you mean by "my latest gradle setup"?
Provide more specific details I will look into this.

**Comment by darkhax** [comment]
Hey there, I made this issue to help a few community of my members hit their goals, but you're more than welcome to take this on. I have a gradle build script which handles a lot of things, and I have made it as cookie cutter as possible. Example [build.gradle](https://github.com/Darkhax-Minecraft/Bookshelf/blob/master/build.gradle) and [gradle.properties](https://github.com/Darkhax-Minecraft/Bookshelf/blob/master/gradle.properties) can be found there.
